### DISCUSSION TITLE
How to immediately update neighbors that we are shutting down

### DISCUSSION THREAD

**Discussion by harshalgohel** [lldpd/lldpd#564] [Q&A] [answered]
Consider this usecase, where multiple raspberry pis are running a some application which is using lldpctl.

Now one of the raspberry pi user decides to go incognito - I want to send last lldpd_shutdown packet and then keep lldpd in receive only / paused mode.

Is this possible with lldpctl?
If not, what is the preferred approach on possible implementation?

**Comment by vincentbernat** [comment, marked as answer]
Not possible yet. I think this should be a new command, similar to `update`.
